lib/gooddata/lcm/actions/synchronize_ldm.rb

```diff
@@ -64,7 +64,8 @@ def call(params)
                 blueprint,
                 update_preference: params.update_preference,
                 execute_ca_scripts: false,
-                exclude_fact_rule: exclude_fact_rule
+                exclude_fact_rule: exclude_fact_rule,
+                only_model: true
```

Comment:
There is a substantial risk of regression here. How are we going to prevent it from happening?

Suggestions:
1) consultation with Services
2) make `only_model` configurable